connectors/sources/salesforce.py

```diff
@@ -1485,6 +1513,12 @@ async def get_docs(self, filtering=None):
         # We collect all content documents and de-duplicate them before downloading and yielding
         content_docs = []
 
+        objects_to_sync = (
```

Comment:
this should also go to `else `statement? I don' think we need it to be top-level